versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx

```diff
@@ -171,16 +203,31 @@ We saw from [Optimization 1](#optimization-1-manage-data-retention) that the eas
 The same can be said for limits. If we only want to keep ~10% of traces to be around more than 14 days, we can set a limit on the maximum
 high retention traces we can keep. That would be `.10 * 7,800,000 = 780,000`.
 
-![](./static/p2bothlimits.png)
+![](./static/p2bothlimits_v2.png)
 
-As we can see, the maximum cost is cut from ~40k per month to ~7.5k per month. This lets us be confident that new
-users on the platform will not accidentally cause cost to balloon.
+As we can see, the maximum cost is cut from ~40k per month to ~7.5k per month, because we no longer allow as many expensive
+data retention upgrades. This lets us be confident that new users on the platform will not accidentally cause cost to balloon.
 
 :::note
 The extended data retention limit can cause features other than traces to stop working once reached. If you plan to
 use this feature, please read more about its functionality [here](../../concepts/usage_and_billing/usage_limits#side-effects-of-extended-data-retention-traces-limit).
 :::
 
+### Set dev/staging limits and view total spent limit across workspaces
+
+Let's say we follow the same logic for our dev and staging environments, which leads us to set limits at 10% of the production
```

Comment:
Following the same logic for our dev...

Let's say is a bit too colloquial too